### PR TITLE
Add a new dataset

### PR DESCRIPTION
**Pull request by jacky26822865**
*Issue #, if available:*

*Description of changes:*
CWB opendata dataset pull requests

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: CWB_opendata [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  - earth
+  - observation
+  - earthquakes
+  - satellite
+  - imagery
+  - weather
+License: http://data.gov.tw/license
+Resources:
+  - Description:
+    ARN:
```
Comment (zflamig) [inline CWB_opendata:17]: Please add the bucket ARN here

File: CWB_opendata [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  - climate
+  - earth
+  - observation
+  - earthquakes
+  - satellite
+  - imagery
+  - weather
+License: http://data.gov.tw/license
+Resources:
+  - Description:
```
Comment (zflamig) [inline CWB_opendata:16]: You can say something like "CWB data lake" for the description

**Review by zflamig — CHANGES_REQUESTED** [review]

**Comment by awsopendata** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: open-data-registry
* Commit ID: 34be1cb113d4442d26439024c471151292b8ebd1
* Result: FAILED
* [Build Logs](https://onc2c36gc4.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=e1459b02-9992-438e-a47f-9d4b92edb202%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by awsopendata** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: open-data-registry
* Commit ID: af1a17f388a179c49807a211a25410a3efa9398a
* Result: FAILED
* [Build Logs](https://onc2c36gc4.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=a39583c1-132a-4b16-9fa1-d0c88321d4ba%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Review by zflamig — APPROVED** [review]

**Pull request merged**